fdbmonitor/fdbmonitor.cpp

```diff
@@ -983,7 +996,7 @@ std::unordered_map<int, std::unordered_set<std::string>> set_watches(std::string
 
 int main(int argc, char** argv) {
 	std::string lockfile = "/var/run/fdbmonitor.pid";
-	std::string _confpath = "/etc/foundationdb/foundationdb.conf";
+	std::string _confpath = "/usr/local/etc/foundationdb/foundationdb.conf";
```

Comment:
It would be better if it was just in one single location I can't see any reasons why not but will have to look at why this is a convention for FreeBSD. That's a great question for the forums I see it a lot in the software which comes down from ports but just don't know exactly 'why'.